pepper-apis/src/test/java/org/broadinstitute/ddp/route/EventServiceTest.java

```diff
@@ -301,16 +386,6 @@ public void testActivityInstantiation_Failure_PreconditionExpressionFalse() thro
         Assert.assertTrue(countAfter == countBefore);
     }
 
-    @Test
-    public void testActivityInstantiation_Failure_CreationExpressionFalse() throws Exception {
```

Comment:
Could you say a few words about where this test has gone?  It's not jumping out at me where it has gone.

Reply:
This has to do with that creation expression, which moved to events.